translations/handleAppTranslations.sh

```diff
@@ -17,6 +17,41 @@ wget https://raw.githubusercontent.com/nextcloud/docker-ci/master/translations/l
 wget https://raw.githubusercontent.com/nextcloud/server/master/build/l10nParseAppInfo.php
 perl ./l10n.pl $1 read
 
+versions='stable11 stable12 master abc'
+
+# build POT files for all versions
+mkdir stable-templates
+for version in $versions
+do
+  # skip if the branch doesn't exist
+  if git branch | egrep "^  $version$" ; then
```

Comment:
Somehow this never works. :/ Works if I extract this to a separate bash script, but not in the docker container. I guess this is because of the `sh` vs `bash` thingy somehow.